SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SchemaGenerator.h

```diff
@@ -24,6 +24,6 @@ void GenerateSubobjectSchemaForActor(FComponentIdGenerator& IdGenerator, UClass*
 	FString SchemaPath, FActorSchemaData& ActorSchemaData, const FActorSchemaData* ExistingSchemaData);
 // Generates schema for a statically attached subobject on an Actor - called by GenerateSubobjectSchemaForActor.
 FActorSpecificSubobjectSchemaData GenerateSchemaForStaticallyAttachedSubobject(FCodeWriter& Writer, FComponentIdGenerator& IdGenerator,
-	FString PropertyName, TSharedPtr<FUnrealType>& TypeInfo, UClass* ComponentClass, const FActorSpecificSubobjectSchemaData* ExistingSchemaData);
+	FString PropertyName, TSharedPtr<FUnrealType>& TypeInfo, UClass* ComponentClass, UClass* ActorClass, int MapIndex, const FActorSpecificSubobjectSchemaData* ExistingSchemaData);
```

Comment:
This function's declaration didn't match the definition (it still worked because it's defined in the same compilation unit as it's used, before it's used)